bindata/network/kuryr/001-rbac.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kuryr
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
+  - nodes
+  - endpoints
+  - services
+  - services/status
+  - pods
+  verbs: ["*"]
```

Comment:
This is... a lot. Do you need the ability to write to services and pods? If so, then that's Ok, but it would be nice to pare this down if possible.

Reply:
Actually we do, Kuryr is using services annotations to differentiate between the k8s side and what has been applied on the OpenStack side

Reply:
Yup, we don't yet use CRD's to keep our information, instead we annotate pods and services. This is because Kuryr was there before CRD's.

Reply:
Gotcha. I'd still like to see the list of verbs made explicit, since the ability to create and delete is almost certainly not necessary.

Reply:
Okay, seems like removing create and delete makes total sense.